### DISCUSSION TITLE
error:INVALID_ARGUMENT: Tensor spin_attr/ntypes_spin:0, specified in either feed_devices or fetch_devices was not found in the GraphINVALID_ARGUMENT: Tensor spin_attr/ntypes_spin:0, specified in either feed_devices or fetch_devices was not found in the Graph

### DISCUSSION THREAD

**Discussion by 9Suyoung9** [deepmodeling/deepmd-kit#2749] [Q&A]
Recently, when I used the Si potential function predicted by Deepmd-kit to calculate the thermal conductivity of lammps, the output file showed "INVALID_ARGUMENT: Tensor spin_attr/ntypes_spin:0,  specified in either feed_devices or fetch_devices was not found in the GraphINVALID_ARGUMENT:  Tensor spin_attr/ntypes_spin:0, specified in either feed_devices or fetch_devices was not found in the Graph.But I don't get this output when I simulate with classical potential functions.

**Comment by wanghan-iapcm** [comment]
Please provide input script of DP training and the input files of your lammps simulation. Without the information we cannot tell what was the problem.

**Comment by njzjz** [comment]
This message is expected and will not terminate the program.
